diff --git a/sites/all/modules/contrib/ctools/plugins/content_types/block/block.inc b/sites/all/modules/contrib/ctools/plugins/content_types/block/block.inc
index 9d55e8a..6a1c5ee 100644
--- a/sites/all/modules/contrib/ctools/plugins/content_types/block/block.inc
+++ b/sites/all/modules/contrib/ctools/plugins/content_types/block/block.inc
@@ -287,7 +287,7 @@ function ctools_block_content_type_admin_info($subtype, $conf) {
   $block = (object) module_invoke($module, 'block_view', $delta);

   // Sanitize the block because <script> tags can hose javascript up:
-  if (!empty($block->content)) {
+  if (!empty($block->content) && is_string($block->content)) {
     $block->content = filter_xss_admin($block->content);
   }
